stubs/PyAutoGUI/pyautogui/__init__.pyi

```diff
@@ -0,0 +1,230 @@
+    y: float | Sequence[int] | None = ...,
+    interval: float = ...,
+    duration: float = ...,
+    tween: Callable[[float], float] = ...,
+    logScreenshot: bool | None = ...,
+    _pause: bool = ...,
+) -> None: ...
+def doubleClick(
+    x: float | Sequence[int] | None = ...,
+    y: float | Sequence[int] | None = ...,
+    interval: float = ...,
+    button: str = ...,
+    duration: float = ...,
+    tween: Callable[[float], float] = ...,
+    logScreenshot: bool | None = ...,
+    _pause: bool = ...,
+) -> None: ...
+def tripleClick(
+    x: float | Sequence[int] | None = ...,
+    y: float | Sequence[int] | None = ...,
+    interval: float = ...,
+    button: str = ...,
+    duration: float = ...,
+    tween: Callable[[float], float] = ...,
+    logScreenshot: bool | None = ...,
+    _pause: bool = ...,
+) -> None: ...
+def scroll(
+    clicks: float,
+    x: float | Sequence[int] | None = ...,
+    y: float | Sequence[int] | None = ...,
+    logScreenshot: bool | None = ...,
+    _pause: bool = ...,
+) -> None: ...
+def hscroll(
+    clicks: float,
+    x: float | Sequence[int] | None = ...,
+    y: float | Sequence[int] | None = ...,
+    logScreenshot: bool | None = ...,
+    _pause: bool = ...,
+) -> None: ...
+def vscroll(
+    clicks: float,
+    x: float | Sequence[int] | None = ...,
+    y: float | Sequence[int] | None = ...,
+    logScreenshot: bool | None = ...,
+    _pause: bool = ...,
+) -> None: ...
+def moveTo(
+    x: float | Sequence[int] | None = ...,
+    y: float | Sequence[int] | None = ...,
+    duration: float = ...,
+    tween: Callable[[float], float] = ...,
+    logScreenshot: bool = ...,
+    _pause: bool = ...,
+) -> None: ...
+def moveRel(
+    xOffset: float | Sequence[int] | None = ...,
+    yOffset: float | Sequence[int] | None = ...,
+    duration: float = ...,
+    tween: Callable[[float], float] = ...,
+    logScreenshot: bool = ...,
+    _pause: bool = ...,
+) -> None: ...
+
+move = moveRel
+
+def dragTo(
+    x: float | Sequence[int] | None = ...,
+    y: float | Sequence[int] | None = ...,
+    duration: float = ...,
+    tween: Callable[[float], float] = ...,
+    button: str = ...,
+    logScreenshot: bool | None = ...,
+    _pause: bool = ...,
+    mouseDownUp: bool = ...,
+) -> None: ...
+def dragRel(
+    xOffset: float | Sequence[int] = ...,
+    yOffset: float | Sequence[int] = ...,
+    duration: float = ...,
+    tween: Callable[[float], float] = ...,
+    button: str = ...,
+    logScreenshot: bool | None = ...,
+    _pause: bool = ...,
+    mouseDownUp: bool = ...,
+) -> None: ...
+
+drag = dragRel
+
+def isValidKey(key: str) -> bool: ...
+def keyDown(key: str, logScreenshot: bool | None = ..., _pause: bool = ...) -> None: ...
+def keyUp(key: str, logScreenshot: bool | None = ..., _pause: bool = ...) -> None: ...
+def press(
+    keys: str | list[str], presses: int = ..., interval: float = ..., logScreenshot: bool | None = ..., _pause: bool = ...
```

Comment:
The docstring says that `keys` has to be `str` or `list[str]`, but it can actually be any iterable of `str`s. And `list[str]` is quite problematic for parameter annotations due to `list` being invariant.

```suggestion
    keys: str | Iterable[str], presses: int = ..., interval: float = ..., logScreenshot: bool | None = ..., _pause: bool = ...
```